surveys/2024-annual-survey/report/main.py

```diff
@@ -0,0 +1,425 @@
+import random
```

Comment:
this file is ... autogenerated, right?

Reply:
xD No, lol, I wrote that myself. It was much easier this year, when I could just copy paste it from last year; last year I had to write the CSV parsing logic + all the charts and then the structure of the report, so it took longer.